simulation.md

```diff
@@ -0,0 +1,454 @@
+
+# Configure DaVinci
+DaVinci().TupleFile = "DVntuple.root"
+DaVinci().Simulation = True
+DaVinci().Lumi = False
+DaVinci().DataType = str(year)
+DaVinci().UserAlgorithms = [printMC, mctuple]
+```
+```shell
+lb-run DaVinci/v41r0 gaudirun.py DaVinciOptions.py
+```
+This script will attempt to build an nTuple from the xgen file it is given, using the specified decay descriptor. If everything is working correctly, this should return one entry per event, corresponding to your signal candidate. The PrintMCTree algorithm will print to screen the full decay chain for each particle in "decay_heads" e.g:
+```
+<--------------------------------- MCParticle --------------------------------->
+                Name         E         M         P        Pt       phi        Vz
+                           MeV       MeV       MeV       MeV      mrad        mm
+B~0                 228676.20   5279.58 228615.24   4575.10   3122.65    -18.05
++-->D-              220232.04   1869.61 220224.11   4909.35  -2995.39     96.91
+|+-->K+             150522.77    493.68 150521.97   3531.59   3092.01    120.50
+|+-->pi-             12700.02    139.57  12699.26    351.94  -2113.03    120.50
+|+-->pi-             57009.20    139.57  57009.03   1290.27  -2667.72    120.50
+|+-->gamma               0.05      0.00      0.05      0.00      0.00    120.50
++-->pi+               8444.15    139.57   8443.00    850.25   1231.87     96.91
+```
+, which is extremely helpful for knowing if your decfile is producing what you think it should. Note that in addition to your signal B0, it will also print out the decay chain for any B0 in the event, so you will regularly see other random B decays.
+
+## DecFiles
+
+```
+# EventType: 11146031
+#
+# Descriptor: [B0 -> (J/psi(1S) -> mu+ mu-)  (K*0 -> K+ pi-) (phi -> K+ K-)]cc
+#
+# NickName: Bd_JpsiphiKst,KKmumuKpi=DecProdCut
+#
+# Cuts: DaughtersInLHCb
+#
+# CPUTime: < 1 min
+#
+# Documentation:  Bd decay to Jpsi(to mu+ mu-), phi(to K+ K-) Kst(K+ pi-) with K+,K-,mu,mu,K+,pi- in acceptance
+# EndDocumentation
+#
+# PhysicsWG: B2Ch 
+# Tested: Yes
+# Responsible: Alessia Satta
+# Email: alessia.satta@cern.ch
+# Date: 20160514
+#
+```
+The information in the header is not just bookkeeping, almost all of it is parsed and changes what you get out at the end. The EventType is a series of flags which controls the generation. The rules for this are described in detail at:
+https://cds.cern.ch/record/855452/files/lhcb-2005-034.pdf
+For example for the first digit of 1 = contains b quark, 2 = c quark, 3 = min bias...
+Similarly, the document specifies the conventions for the "NickName" - which also has to be the filename. Note that once MC has been produced from a given DecFile, it is not allowed to be changed, so you never need to worry about which version of DecFiles you are looking at when trying to understand existing samples.
+
+The "Cuts" field specifies one of a set of C++ selections in:
+https://gitlab.cern.ch/lhcb/Gauss/blob/master/Gen/GenCuts/
```

Comment:
🔗 maybe:

"The "Cuts" field specifies which 'cut tool' to use out of a predefined set. The [source code](https://gitlab.cern.ch/lhcb/Gauss/blob/master/Gen/GenCuts/) of the tool you're using should be consulted to fid out what it does."